spec/lib/open_project/custom_field_format_spec.rb

```diff
@@ -41,47 +41,55 @@
     end
 
     context "for a 'Project' class" do
-      it_behaves_like "custom field formats",
-                      "Project",
-                      ["bool", "date", "float", "int", "link", "list", "string", "text", "user", "version"]
+      context "with calculated values feature flag enabled", with_flag: { calculated_value_project_attribute: true } do
+        it_behaves_like "custom field formats",
+                        "Project",
+                        %w[bool calculated_value date float int link list string text user version]
+      end
+
+      context "with calculated values feature flag disabled" do
```

Comment:
From experience of removing feature flag, it is nicer when disabled flag is explicit:

```suggestion
      context "with calculated values feature flag disabled", with_flag: { calculated_value_project_attribute: false } do
```